features/FEATURE_BLE/targets/TARGET_NORDIC/TARGET_NORDIC_SOFTDEVICE/mbed_lib.json

```
@@ -40,6 +40,10 @@
             "help": "Sets the maximum number of descriptors that can be registered",
             "value": "8",
             "macro_name": "NRF_SDH_BLE_TOTAL_DESCRIPTORS"
+        },
+        "gatt_char_vloc_location": {
```

Cryptic name IMHO (also vloc already has location in the name). Seeing as it's boolean maybe:
```suggestion
        "user_managed_gatt_characteristic_memory": {
```